src/command-line/index.js

```diff
@@ -51,11 +51,9 @@ Helper.setHome(home);
 
 require("./start");
 require("./config");
-require("./list");
-require("./add");
-require("./remove");
-require("./reset");
-require("./edit");
+if (!Helper.config.public && !Helper.config.ldap.enable) {
```

Comment:
The only problem here is that CLI overrides (e.g. `--public` and later on `--config` will have no effect on this.

Reply:
That's a good point.
A bit niche right now I think though, because the only situation where it could hurt is if your config file says `public: true` but you intend to run `thelounge --private` and try to run `thelounge add <user>`. That seems very specific and unlikely to me (as opposed to port and stuff, or even run the other way around: private in config file and `--public`).

One solution I can see is moving `-c` at `thelounge` level. So `thelounge -c public=false start`. That way, you can do `thelounge -c public=false add <user>` and it would work. What do you think?
In general, it might be better to do that anyway, because in a world where we have package, we'll want to give the ability to packages to register their own commands, and they would benefit from overrides just as much.
I'm starting to really prefer this idea actually.

Reply:
In your other PR, just do `parseConfigOptions` after doing `setHome` in `index.js` and then it will work as you are describing.

Reply:
Yes exactly. I just wanted to make sure you were on board with that. It's a breaking change from what you describe, but such a niche one that it doesn't matter, and `-c` for 2.7.0 brings way more benefits.

Only thing is `thelounge start -c public=false` will not work anymore, options have to be passed before start (I could duplicate in `index.js` AND `start.js` but that starts to be too intricate).